File: connecttoapilibrary/src/main/java/com/sec/connecttoapilibrary/onlinefm/liveRadioData/PropertyInfo.java
package com.sec.connecttoapilibrary.onlinefm.liveRadioData;

import java.io.Serializable;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/1/29.
 */

public class PropertyInfo implements Serializable {

    private int id;
    private String name;

    public void setPropertyInfoId(int id){
        this.id = id;
    }
    public void setPropertyInfoname(String name){
        this.name = name;
    }
    public int getPropertyInfoId(){
        return id;
    }
    public String getPropertyInfoname(){
        return name;
    }

    public static class values {
        private int id;
        private String name;
        private String sequence;
        public void setvaluesId(int id){
            this.id = id;
        }
        public void setvaluesname(String name){
            this.name = name;
        }
        public void setvaluessequence(String sequence){
            this.sequence = sequence;
        }
        public int getvaluesId(){
            return id;
        }
        public String getvaluesname(){
            return name;
        }
        public String getvaluessequence(){
            return sequence;
        }
    }
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/data/PropertyInfo.kt
package com.sec.myonlinefm.data


/**
 * Created by SRC-TJ-MM-BinYang on 2018/1/29.
 */
class PropertyInfo {
    private var id = 0
    private var name: String? = null
    fun setPropertyInfoId(id: Int) {
        this.id = id
    }

    fun setPropertyInfoname(name: String?) {
        this.name = name
    }

    fun getPropertyInfoId(): Int {
        return id
    }

    fun getPropertyInfoname(): String? {
        return name
    }

    class values {
        private var id = 0
        private var name: String? = null
        private var sequence: String? = null
        fun setvaluesId(id: Int) {
            this.id = id
        }

        fun setvaluesname(name: String?) {
            this.name = name
        }

        fun setvaluessequence(sequence: String?) {
            this.sequence = sequence
        }

        fun getvaluesId(): Int {
            return id
        }

        fun getvaluesname(): String? {
            return name
        }

        fun getvaluessequence(): String? {
            return sequence
        }
    }
}